lib/logstash/filters/ruby.rb

```diff
@@ -43,7 +43,11 @@ def filter(event,&block)
       @codeblock.call(event,&block)
       filter_matched(event)
     rescue Exception => e
-      @logger.error("Ruby exception occurred: #{e}")
+      err_details = {}
+      err_details["exception"] = e
+      err_details["event"] = event
```

Comment:
If you have different kinds of exceptions happening on the same code block you wont be able to know what kind of events are triggering each one.
Maybe a boolean verbose configuration option that defaults to false? When true the event is shown, or maybe just event["message"].
What do you think?